CHANGELOG.md

```diff
@@ -7,6 +7,22 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 <!-- towncrier release notes -->
 
+## [v0.96.0-sumo-1]
+
+### Released 2024-03-20
+
```

Comment:
I think that it is worthy to add information about known issue also to this release
https://github.com/SumoLogic/sumologic-otel-collector/blob/main/CHANGELOG.md#known-issue